src/interfaces.ts

```diff
@@ -81,6 +81,13 @@ export interface ObfuscatedShard {
   ranges: Range[];
 }
 
+export interface BanditFlagAssociation {
```

Comment:
`BanditVariation` is the name I've seen for this class elsewhere
~~faint whispers of "protobufs" come from off in the distance.

Reply:
Oh yeah happy to use that name!